README.md

```diff
@@ -35,7 +35,7 @@ Application Options:
       --file=sql_file               Read schema SQL from the file, rather than stdin (default: -)
       --dry-run                     Don't run DDLs but just show them
       --export                      Just dump the current schema to stdout
-      --skip-drop                   Skip destructive changes such as DROP
+      --enable-drop                 Ebale destructive changes such as DROP
```

Comment:
I'm sorry that my comments in https://github.com/k0kubun/sqldef/issues/292 were confusing, but can we call it `--enable-drop-table`? It'd be difficult to correctly guess the actual behavior from `--enable-drop`.

It'd be also nice to update the description on the right column accordingly. `Ebale` looks like a typo.

Reply:
I think so.
I modify `enable-drop` to `enable-drop-table`.